lib/engines/content_block_manager/app/components/content_block_manager/content_block/document/show/document_timeline/timeline_item_component.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="timeline__item">
+  <div class="timeline__header">
+    <% if is_latest %>
+      <span class="timeline__latest">Latest</span>
+      <br>
+    <% end %>
+    <h2 class="timeline__title"><%= title %></h2>
+    <p class="timeline__byline">by <%= byline %></p>
+  </div>
+
+  <p class="timeline__date">
+    <%= date %>
+  </p>
+
+  <% if version.field_diffs.present? %>
+    <div class="timeline__diff-table">
+      <%= render "govuk_publishing_components/components/details", {
+        title: "Details of changes",
+        open: is_latest,
+      } do %>
+        <% capture do %>
```

Comment:
Do we need this `capture` block? Looks like we're not assigning it to anything, and we're only calling one method inside the block so if we needed the string we could just call the method?

Reply:
Ah, just tried that, and it renders outside the details container unfortunately 😞